### ISSUE TITLE
Update pagination sr-only heading levels

### ISSUE THREAD

**Issue by danahertzberg** [az-digital/az_quickstart#2718]
## Problem/Motivation
I want to meet accessibility standards

### Describe the bug

Pagination has a sr-only heading that should be `<h3>` tag

> [Page Numbers and Heading Level](https://forge.arizona.edu/news/general-news)
>  
> DubBot is complaining about the Pagination heading that is on the page. This heading has a class of sr-only (screen reader only) and as I have been told this is not showing on screen but screen readers can see the text.
>  
> The reason DubBot caught this was because they jumped from heading level 2 to 4 in their content. They will address this with moving their news items to heading level 3’s. Is this by design to have this pagination heading with a class of sr-only in Quick Start and hard coding it to a heading level 4? If so, can its heading level be controlled in the future to potentially resolve this gap in cases where content can’t meet the heading level requirements here?

## Proposed resolution
Description of the proposed solution, the rationale behind it, and workarounds for people who cannot use the patch.

##  Screenshots
If applicable, add screenshots to help explain your problem. (Need a tool to share screenshots or screencasts as URLs? Try [monosnap](https://monosnap.com)
